src/main/java/lz/modelTesting/controllers/ProjectsController.java

```diff
@@ -58,14 +58,23 @@ public ProjectsController(ProjectsRepository projectsRepository, ObjectMapper ob
     @PostMapping(value = "/add")
     public ResponseEntity<String> addProject(HttpServletRequest request) {
         Project project = createProjectFromRequest(request);
+        projectsRepository.save(project);
         String id = project.getId();
         return ResponseEntity.ok().body(id);
     }
 
-    private Project createProjectFromRequest(HttpServletRequest request) {
+    @PostMapping(value = "/changename/{id}")
```

Comment:
Change name seems like PUT method, not POST.
It's better to write `change-name` instead of `changename`